jest.config.ts

```diff
@@ -1,12 +1,34 @@
 import { Jest } from './src';
 
-export default Jest.mergePreset({
+const defaultConfig = Jest.mergePreset({
   coveragePathIgnorePatterns: ['<rootDir>/integration/', '<rootDir>/template/'],
   displayName: 'unit',
   setupFiles: ['<rootDir>/jest.setup.ts'],
-  testPathIgnorePatterns: [
-    '<rootDir>/template/',
-    '\\.int\\.test\\.ts',
-    '/test\\.ts',
+  testPathIgnorePatterns: ['<rootDir>/template/', '/test\\.ts'],
+  watchPathIgnorePatterns: [
+    '<rootDir>/integration/format/',
+    '<rootDir>/integration/lint/',
   ],
 });
+
+const { transform, moduleNameMapper, setupFiles } = defaultConfig;
+
+export default {
+  ...defaultConfig,
+  projects: [
+    {
+      transform,
+      moduleNameMapper,
+      setupFiles,
+      displayName: 'unit',
+      testPathIgnorePatterns: ['<rootDir>/template/', '\\.int\\.test\\.ts'],
+    },
```

Comment:
Would it be too heavy handed for `Jest.mergePreset` to provide the root defaults to each project, like e.g.

```typescript
projects: projects.map(project => ({ transform, moduleNameMapper, ...project })
```

Reply:
Yeah I wasn't sure, I think it makes sense though